modules/cnf-scheduling-workload-onto-worker-with-real-time-capabilities.adoc

```diff
@@ -0,0 +1,187 @@
+The label selectors must match the nodes that are attached to the machine config pool that was configured for low latency by the Node Tuning Operator.
+====
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in as a user with `cluster-admin` privileges.
+
+* You have applied a performance profile in the cluster that tunes worker nodes for low latency workloads.
+
+.Procedure
+
+. Create a `Pod` CR for the low latency workload and apply it in the cluster, for example:
++
+.Example `Pod` spec configured to use real-time processing
+[source,yaml,subs="attributes+"]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dynamic-low-latency-pod
+  annotations:
+    cpu-quota.crio.io: "disable" <1>
+    cpu-load-balancing.crio.io: "disable" <2>
+    irq-load-balancing.crio.io: "disable" <3>
+spec:
+  securityContext:
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
+  containers:
+  - name: dynamic-low-latency-pod
+    image: "registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version}"
+    command: ["sleep", "10h"]
+    resources:
+      requests:
+        cpu: 2
+        memory: "200M"
+      limits:
+        cpu: 2
+        memory: "200M"
+    securityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop: [ALL]
+  nodeSelector:
+    node-role.kubernetes.io/worker-cnf: "" <4>
+  runtimeClassName: performance-dynamic-low-latency-profile <5>
+# ...
+----
+<1> Disables the CPU completely fair scheduler (CFS) quota at the pod run time
+<2> Disables CPU load balancing
+<3> Opts the pod out of interrupt handling on the node
+<4> The `nodeSelector` label must match the label that you specify in the `Node` CR.
+<5> `runtimeClassName` must match the name of the performance profile configured in the cluster.
+
+. Enter the pod `runtimeClassName` in the form performance-<profile_name>, where <profile_name> is the `name` from the `PerformanceProfile` YAML, in this example, `performance-dynamic-low-latency-profile`.
+
+. Ensure the pod is running correctly. Status should be `running`, and the correct cnf-worker node should be set:
++
+[source,terminal]
+----
+$ oc get pod -o wide
+----
++
+.Expected output
++
+[source,terminal]
+----
+NAME                     READY   STATUS    RESTARTS   AGE     IP            NODE          NOMINATED NODE   READINESS GATES
+dynamic-low-latency-pod  1/1     Running   0          5h33m   <ip-address>  <node-name>   <none>           <none>
+----
+
+. Get the CPUs that the pod configured for IRQ dynamic load balancing runs on:
++
+[source,terminal]
+----
+$ oc exec -it dynamic-low-latency-pod -- /bin/bash -c "grep Cpus_allowed_list /proc/self/status | awk '{print $2}'"
+----
++
+.Expected output
++
+[source,terminal]
+----
+Cpus_allowed_list:  2-3
+----
+. Ensure the node configuration is applied correctly. Log in to the node to verify the configuration.
```

Comment:
Add .Verification header. 


And, maybe
```suggestion
* Ensure the node configuration is applied correctly: 

.. Log in to the node to verify the configuration.
```
And turn all of the steps into substeps?